core/iwasm/interpreter/wasm_mini_loader.c

```diff
@@ -3233,8 +3233,11 @@ load(const uint8 *buf, uint32 size, WASMModule *module,
 }
 
 WASMModule *
-wasm_loader_load(uint8 *buf, uint32 size, const LoadArgs *args, char *error_buf,
-                 uint32 error_buf_size)
+wasm_loader_load(uint8 *buf, uint32 size,
+#if WASM_ENABLE_MULTI_MODULE != 0
+                 bool /* main_module */,
```

Comment:
Is it better to use `bool main_module`, not sure whether using `bool` without argument name will cause compiler warning/error?

Reply:
I'm not aware of any compilers that will warn for an unnamed parameter. However, if we gave it a name, then all compilers are capable of warning about unused named parameters, so we would have to add another block inside the function body inside the function like this:

```
#if WASM_ENABLE_MULTI_MODULE != 0
                 (void)main_module;
#endif
```

This is fine with me if you prefer it, just let me know.

Reply:
Yes, maybe that's better, could you help update the code? Thanks.

Reply:
@bnason-nf Since we planning to create a new release and will do more tests, I updated your code manually and commit it.